Solutions/Corvus.SpecFlow.Extensions.Specs/Containers/NoContainer.feature

```diff
@@ -0,0 +1,8 @@
+﻿Feature: NoContainer
+	In cases where my tests don't need a DI container
+	As a developer
+	I don't want a container to be set up unless I asked for it
+
+Scenario: Container not present
+	When I call ContainerBindings.GetServiceProvider inside a try block
```

Comment:
Is "inside a try block" really fundamental to the spec? Surely it doesn't matter whether you call it inside a try block or not, it's still expected to throw the exception.

Reply:
It is not fundamental to the spec, but it is a necessary part of the construction of this spec. This is one of (many) symptoms of the trickiness of using a SpecFlow test to test some extensions designed to use used as part of a SpecFlow test...

I had avoided just "When I call ContainerBindings.GetServiceProvider" because I thought I wanted it to be clear from the step name that the step was going to have to be ready for failure. But on reflection, I don't think I care about that, so I've gone with a simpler name now.